Packs/SplunkPy/Integrations/SplunkPy/SplunkPy_test.py

```diff
@@ -2633,6 +2633,37 @@ def test_basic_authentication_param(mocker, username, expected_username, basic_a
     assert ('basic' in client.connect.call_args[1]) == basic_auth
 
 
+@pytest.mark.parametrize(argnames='host, expected_base_url', argvalues=[
+    ('8.8.8.8', 'https://8.8.8.8:8089/'),
+    ('https://www.test.com', 'https://www.test.com:8089/'),
+    ('http://www.test.com', 'https://http://www.test.com:8089/'),  # we don't want to silently replace http with https
```

Comment:
For these cases why not raise an appropriate error like "Server URL must use SSL (https) ..." or something like this?